### PR TITLE
PLANET-7127: Apply the new Greenpeace sans typeface

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: [PLANET-7127](https://jira.greenpeace.org/browse/PLANET-7127)

## Description
Apply the new Greenpeace Sans (primary) typeface to:
- Headings
- Buttons
- Navigation menu items

Related to
 - #2008 
 - #2051
 - #2052
 - https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1078

### CODE REVIEW

File: assets/src/scss/base/_tokens.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /* CSS Variables */
 :root {
   /* Primitives */
-  --font-family-greenpeace-sans: "GreenpeaceSans";
+  --font-family-greenpeace-sans: "GreenpeaceSans-Bold";
```
Comment (GP-Dan-Tovbein) [inline assets/src/scss/base/_tokens.scss:6]: We should review this token

File: assets/src/scss/new-identity/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+$bucket: "https://www.greenpeace.org/static/planet4-assets/";
```
Comment (GP-Dan-Tovbein) [inline assets/src/scss/new-identity/_fonts.scss:1]: This implementation is not part of this ticket. It's added within #2008

File: assets/src/scss/new-identity/style.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @import "../base/tokens";
 @import "../base/variables";
 @import "../base/mixins";
+@import "./fonts";
```
Comment (GP-Dan-Tovbein) [inline assets/src/scss/new-identity/style.scss:4]: This implementation is not part of this ticket. It's added within https://github.com/greenpeace/planet4-master-theme/pull/2008

**Comment by comzeradd** [comment]
Assets build fails on this one:

```
ERROR in ./assets/src/scss/partials/navigation-bar-light.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Cannot generate prefix when using multiple selectors. .nav-item _--,.nav-donate _--
```

Maybe we need to upgrade dash-dash?

**Review by comzeradd — APPROVED** [review]

**Pull request merged**